src/views/open-api-docs-view-v2/utils/slugify-operation-id.ts

```diff
@@ -0,0 +1,21 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import slugify from 'slugify'
+
+/**
+ * Given an operation ID, return a slugified version of it.
+ *
+ * This function has been abstracted as it's important for all instances
+ * of slugify-ing operation IDs to function in exactly the same way,
+ * as they're used for generating and then later matching URLs for
+ * individual operation pages.
+ *
+ * @param operationId
+ * @returns
+ */
+export function slugifyOperationId(operationId: string): string {
```

Comment:
I split this out as it's important for all instances of slugify-ing operation IDs to function in exactly the same way, as they're used to match URLs to the appropriate operation.